### PR TITLE
Component updates

### PR DESCRIPTION
**Pull request by heoelri**

### CODE REVIEW

**Comment by msimecek** [comment]
Shall we run e2e and merge? Before it drifts from main?

**Comment by nielsams** [comment]
> We've still three grafana PRs open #69, #70 and #71. @nielsams do you want to address them separately or does it make sense to merge them into `component-updates` and test it here?

I will test them separately this morning

**Comment by heoelri** [comment]
Validating the current state here: https://dev.azure.com/alwayson/Foundational-Online/_build/results?buildId=303&view=results

File: src/infra/monitoring/grafana/healthmodelpanel/package.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
   "license": "Apache-2.0",
   "devDependencies": {
     "@grafana/data": "^8.2.2",
-    "@grafana/toolkit": "^8.3.4",
+    "@grafana/toolkit": "^8.3.5",
```
Comment (sebader) [inline src/infra/monitoring/grafana/healthmodelpanel/package.json:17]: shouldnt this not be updated? (and the healthmodelpanel/yarn.lock)

Comment (heoelri) [reply src/infra/monitoring/grafana/healthmodelpanel/package.json:17]: this is already the previous update #69, #70 and #71 contain already the next update

Comment (heoelri) [reply src/infra/monitoring/grafana/healthmodelpanel/package.json:17]: `8.3.5` to `8.3.6`

Comment (heoelri) [reply src/infra/monitoring/grafana/healthmodelpanel/package.json:17]: we need a way to properly test that in e2e now that grafana is in main

**Pull request merged**